recipes/ccfits/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ccfits" %}
+{% set version = "2.5" %}
+{% set sha256 = "e542f39fa496417c401be7fbca715d566462cf4c56c83e3d43b6df3cb8d92105" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: CCfits-{{version}}.tar.gz
+  url: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/CCfits-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+ 
+ build:
+     - toolchain
+     - gcc
+     - cfitsio
+     - cmake
+
+ run:
+     - cfitsio
+     - libgcc
+   
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libCCfits.so  # [linux]
+    
+    - test -f ${PREFIX}/lib/libCCfits.dylib  # [osx]
+    
+    - test -d ${PREFIX}/include/CCfits
+    
+    - conda inspect linkages -p $PREFIX ccfits  # [not win]
+    - conda inspect objects -p $PREFIX ccfits  # [osx]
+
+   
+about:
+  summary: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. It is written in ANSI C++ and implemented using the C++ Standard Library with namespaces, exception handling, and member template functions.
+  home: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/
+  license: GPL2
```

Comment:
Please include the `license_file` and the `license_family`.

Reply:
Thanks, I just did.